plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -212,7 +212,13 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
                EOS_ASSERT(false, plugin_exception, "");
             }
 #endif
-         }), "Override default WASM runtime")
+         }), "Override default WASM runtime(\"eos-vm\", \"eos-vm-jit\", \"eos-vm-oc\").\n"
+         "\"eos-vm\" : A WebAssembly interpreter providing deterministic and efficient time bound execution with debugging support.\n"
+         "\"eos-vm-jit\" : A WebAssembly compiler that takes WASM and generates native code on the fly.\n"
```

Comment:
A WebAssembly compiler that takes WASM and generates native code using a just in time compitler.

Reply:
changed